docs/Materials.md.html

```diff
@@ -1622,6 +1623,12 @@
     To obtain API-level world space coordinates, custom materials should add `getWorldOffset()` to
     `getWorldPosition()` (et al).
 
+!!! TIP: sampling from render targets
+    When writing materials for the surface domain, please use `sampleRenderTarget2D` instead of
```

Comment:
Did you mean `post process` instead of `surface`?

Reply:
I actually mean `surface` here.  In a post-process shader, the uv we provide is already flipped.  In the surface domain, we do not provide uv's so we provide this helper instead.

Reply:
I see, that's a little weird. I wonder if instead we shouldn't provide a method to flip the UVs: `uvToRenderTargetUV()` and just use `texture()` as usual.

Reply:
I'm fine with that idea, it will be more verbose but it will also be more flexible.